docs/src/main/sphinx/security/cli.rst

```diff
@@ -90,17 +90,21 @@ Additional Kerberos debugging information
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can enable additional Kerberos debugging information for the Trino CLI
-process by passing ``-Dsun.security.krb5.debug=true`` as a JVM argument, when
-starting the CLI process. Doing so requires invoking the CLI JAR via ``java``
-instead of running the self-executable JAR directly. The self-executable jar
-file cannot pass the option to the JVM.
+process by passing ``-Dsun.security.krb5.debug=true``,
+``-Dtrino.client.debugKerberos=true``, and
+``-Djava.security.debug=gssloginconfig,configfile,configparser,logincontext``
```

Comment:
How about `-Djava.security.debug=all`? That would be more future proof.

Reply:
I tried to only pick options relevant to Kerberos. Turning on all of them adds a ton of messages about TLS certificates being loaded, which makes debugging Kerberos harder: https://docs.oracle.com/javase/8/docs/technotes/guides/security/troubleshooting-security.html

Reply:
Should we include this link in the docs?

Reply:
I think we can mention the link, but it should not point to java 8. Use java 11 at least.

Reply:
Of course, this was the first result from Google :D The pages for Java 11 look a lot more modern, I linked these two:
* https://docs.oracle.com/en/java/javase/11/security/troubleshooting-security.html
* https://docs.oracle.com/en/java/javase/11/security/troubleshooting.html

These docs are pretty comprehensive and I think it's very good to reference them.